### PR TITLE
docs(examples): Add Remix Auth with GitHubStrategy example

### PR DESCRIPTION
**Pull request by sergiodxa**
This adds an example using [Remix Auth](https://github.com/sergiodxa/remix-auth) with the [GitHubStrategy](https://github.com/sergiodxa/remix-auth-github).

### CODE REVIEW

File: examples/remix-auth/github/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+BASE_URL=http://localhost:3000/
```
Comment (kentcdodds) [inline examples/remix-auth/github/.env.example:3]: I'd prefer that this be derived from the request at runtime like so: https://github.com/kentcdodds/kentcdodds.com/blob/main/app/utils/misc.tsx#L229

File: examples/remix-auth/github/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Remix Auth - GitHubStrategy
+
+Authentication using Remix Auth with the GitHubStrategy.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth/github)
```
Comment (kentcdodds) [inline examples/remix-auth/github/README.md:9]: Let's make this flat after all

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-github)
```

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Super! Just a couple things

File: examples/remix-auth/github/.env.example (hunk 1 above)
Comment (sergiodxa) [reply examples/remix-auth/github/.env.example:3]: That should need to be supported by the strategy code, this needs to be initialized outside the loader

**Comment by sergiodxa** [comment]
@kentcdodds I moved the code from remix-auth/github to remix-auth-github, I also moved the code of remix-auth/form to remix-auth-form

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**